cli/src/main/java/io/codiga/cli/utils/DatadogUtils.java

```diff
@@ -149,4 +139,26 @@ public static List<AnalyzerRule> getRulesFromDatadog(Configuration configuration
         }
         return result;
     }
+
+    /**
+     * Get a Datadog credential from Environment Variables using both prefixes `DATADOG_` and `DD_`.
+     * If none variable is set, the process exists with status=2.
+     * If both variables are set, the variable that has the `DATADOG_` prefix is used.
+     * @param datadogKey The environment variable key with the `DATADOG_` prefix.
+     * @param ddKey The environment variable key with the `DD_` prefix.
+     * @return The credential
+     */
+    private static String getCredential(String datadogKey, String ddKey) {
```

Comment:
Nit: name the variables `variableName` and `fallbackVariableName` as it shows one has the priority over the other. Not blocking.